README.md

```diff
@@ -123,55 +55,31 @@ Collect the data through teleoperation.
   ```
 Refer to [official hello robot keybindings](https://docs.hello-robot.com/0.2/stretch-tutorials/getting_started/images/xbox.png) to learn how to operate Stretch with the controller. We also provide a `Dataloaders` to load data into torch (WIP). 
 
-### Launching Grasping Demo (outdated)
-
-You need to create a catkin workspace on your workstation in order to run this demo, as this is where we will run [Contact Graspnet](https://github.com/cpaxton/contact_graspnet/tree/cpaxton/devel).
-
-Contact graspnet is downloaded under `third_party/`, but there is a `CATKIN_IGNORE` file in this directory. You want to symlink this file out into your workspace:
-```
-ROSWS=/path/to/ros_ws
-ln -s `rospack find home_robot`/third_party/contact_graspnet $ROSWS/src/contact_graspnet
-```
-... but it actually shouldn't be necessary. What is necessary is to build the grasp service defined in `home_robot` by placing it into `$ROSWS`.
-
+### Syncing code between Robot and Workstation
 
-Put the robot in its initial position, e.g. so the arm is facing cups you can pick up. On the robot side:
+Let `ROBOT_IP` store the robot's IP and let `WORKSTATION_IP` store the workstation's IP. If your local network doesn't have access to internet we recommend using `rsync` with `--update` flag to sync your code changes across the machines. Usage:
 ```
-roslaunch home_robot startup_stretch_hector_slam.launch
+rsync -rvu /abs/path/to/local/home-robot $ROBOT_USER@$ROBOT_IP:/abs/path/to/remote/home-robot
 ```
 
-#### Note: Contact GraspNet
-
-Contact graspnet is supported as a way of generating candidate grasps for the Stretch to use on various objects. We have our own fork of [Contact Graspnet](https://github.com/cpaxton/contact_graspnet/tree/cpaxton/devel) which has been modified with a ROS interface.
+The above command will do a *r*ecursive *u*pdating of changed files while echoing a *v*erbose output.
 
-Follow the installation instructions as normal and start it with:
-```
-conda activate contact_graspnet_env
-~/src/contact_graspnet$ python contact_graspnet/graspnet_ros_server.py  --local_regions --filter_grasps
-```
 
-### Troubleshooting 
+## Troubleshooting 
 
 - `ImportError: cannot import name 'gcd' from 'fractions'`: Launch ros nodes from an env with Python 3.8 instead of 3.9
```

Comment:
Is this valid? i run ROS nodes from 3.10 all the time

Reply:
I've been using 3.9 without this issue with the current environment files. Maybe I should just remove this.

Reply:
yeah i think just remove it for now